components/molecules/VCopyright/VCopyright.vue

```diff
@@ -1,79 +1,81 @@
 <script setup lang="ts">
 import type { PropType } from 'vue'
-import type { Placement } from 'floating-vue'
-import { Tooltip as VTooltip } from 'floating-vue'
+import type { Placement, Tooltip } from 'floating-vue'
 
-defineProps({
+import { hash } from 'ohash'
+
+const props = defineProps({
     placement: {
         type: String as PropType<Placement>,
         default: 'top-end',
     },
-    content: { type: String },
+    content: String,
+    container: {
+        type: String,
+        default: 'body',
+    },
 })
 
-const id = useId()
+// Workaround for https://github.com/Akryum/floating-vue/issues/1006
+const id = computed(() => hash(props.content))
+
+const tooltipProps = computed(() => {
+    return {
+        distance: '0',
+        skidding: '-8',
+        placement: props.placement,
+        triggers: ['hover', 'focus', 'click'],
+        container: props.container,
+        delay: { show: 0, hide: 100 },
+        ariaId: id.value,
+        handleResize: true,
+        eagerMount: true,
+    }
+})
+
+const displayedComponent = shallowRef<string | typeof Tooltip>('div')
+const isTooltipComponentDisplayed = computed(() => displayedComponent.value !== 'div')
+
+async function onUserInteract() {
+    if (isTooltipComponentDisplayed.value) return
+
+    import('assets/scss/vendors/_floating-vue.scss')
```

Comment:
why importing scss file on runtime? The css file needed is included into the CSS vendors. Should we import the file here?

Reply:
the scss file will be compilated in a css file on runtime